modules/rostests/winetests/winhttp/winhttp.c

```diff
@@ -4276,7 +4276,15 @@ static void test_IWinHttpRequest(int port)
 
     hr = IWinHttpRequest_get_ResponseText( req, &response );
     ok( hr == S_OK, "got %08x\n", hr );
+#if __REACTOS__
```

Comment:
So the problem is that sometimes the buffer is valid but its length is less than 32 WCHARs, therefore the dbgstr overruns the buffer?

Would it make sense to do this, then?
`wine_dbgstr_wn(response, min(SysStringByteLen(response) / sizeof(WCHAR), 32))`

Reply:
Normally the size in bytes returned here is '209408' or '0'(error), but admittedly your suggestion is a definite improvement. See what you think about the current changes. Thanks.